public/doc/swagger-2.yaml

```diff
@@ -299,6 +273,30 @@ paths:
         '422':
           $ref: '#/responses/InvalidEntity'
   '/portfolio_items/{portfolio_item_id}':
+    get:
+      tags:
+        - users
+        - admins
+      summary: Fetch a specific Portfolio Item
+      operationId: fetchPortfolioItemWithId
+      description: >
+        By passing in the portfolio id and portfolio_item_id you can fetch the
```

Comment:
@lindgrenj6 - This can be changed to not include information about the portfolio id.